Makefile.libcompat

```diff
@@ -73,7 +73,7 @@ LIBCOMPATIMAKE+=	${LIBCOMPATWMAKE:NINSTALL=*:NDESTDIR=*} \
 .if ${MK_FILE} != "no"
 _libmagic=	lib/libmagic
 .endif
-.if ${MK_PMC} != "no" && (${TARGET_ARCH} == "amd64" || ${TARGET_ARCH} == "morello")
+.if ${MK_PMC} != "no" && (${TARGET_ARCH} == "amd64" || ${TARGET_ARCH:Maarch64*})
```

Comment:
This is a change of behaviour, but if both purecap and hybrid libpmc work then that's actually correct, and if so it can be written more nicely as:
```suggestion
.if ${MK_PMC} != "no" && (${TARGET_ARCH} == "amd64" || ${TARGET_CPUARCH} == "aarch64")
```
otherwise to get the old behaviour you need to add a `c*` on the end of your glob.

Reply:
The change of behavior was needed to fix the libpmc build for hybrid libcompat.